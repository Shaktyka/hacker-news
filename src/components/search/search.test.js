import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import Search from './search.js';

describe('Search', () => {

  it(`Search renders correctly`, () => {
  	const div = document.createElement('div');
  	ReactDOM.render(
  	  <Search>Поиск</Search>,
  	  div
  	);
  	ReactDOM.unmountComponentAtNode(div);
  });

  test('есть корректный снимок', () => {
    const component = renderer.create(
      <Search>Поиск</Search>
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
