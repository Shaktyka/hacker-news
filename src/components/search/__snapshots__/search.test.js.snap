// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search Search: есть корректный снимок 1`] = `
<form>
  <input
    type="search"
  />
  <button
    type="submit"
  />
</form>
`;

exports[`Search есть корректный снимок 1`] = `
<form>
  Поиск
  <input
    type="search"
  />
  <button
    type="submit"
  >
    Поиск
  </button>
</form>
`;
